components/profile/helperFunctions.tsx

```diff
@@ -0,0 +1,48 @@
+import { Dispatch, SetStateAction } from "react";
+import { UserInfoType } from "../../lib/types/HelperTypes";
+
+export const setIsValidURL = async (url: string, updater: Dispatch<SetStateAction<any>>, data: any) => {
```

Comment:
any? I would say, whenever you don't know a type, you should place a comment like "//TODO: remove any", what do you think?

Reply:
Changed. This had to be UserInfoType like the function below